src/runtime_src/tools/xclbinutil/SectionIPLayout.cxx

```diff
@@ -119,6 +119,54 @@ SectionIPLayout::getIPControlType(std::string& _sIPControlType) const
   throw std::runtime_error(errMsg);
 }
 
+// ip_layout -> ip_data -> ps_Kernel -> m_functional (xclbin.h)
+const std::string
+SectionIPLayout::getFunctionalStr(FUNCTIONAL _eFunctional) const
+{
+  switch (_eFunctional) {
+    case FC_DPU:
+      return "DPU";
+    case FC_PREPOST:
+      return "PrePopst";
```

Comment:
Looking over the code base, it looks like there isn't test cases that uses this value, hence it wasn't caught.

Could you also add a new entry in one of the JSON files that has this value.

Reply:
testcase added